src/main/java/kafka/ReactiveKafkaClient.java

```diff
@@ -145,7 +149,12 @@ public void dispose() {
         @Override
         public void run() {
             try {
-                consumer.subscribe(topics, consumerRebalanceListener);
+                List<String> patterns = new ArrayList<>();
+                topics.forEach(topic -> patterns.add(String.format("^%s$", topic)));
```

Comment:
use `stream` like you did [here](https://github.com/osskit/dafka-consumer/pull/15/files#diff-984c8441b3c42bc5908689bf5f13fa2e9b859487fbfa18eff1207570075207acR17)